Updated handbook/company/why-this-way#why-read-documentation (#19076)
fleetdm/fleet: handbook/company/why-this-way.md

Everyone [can contribute](https://fleetdm.com/handbook/company#openness) to Fleet's documentation.  Here are a few principles to keep in mind:
 
- **🔌 Start simple.** It's easier to learn when you aren't overwhelmed.  Good documentation pages and sections start _prescriptive, brief, and clear_; ideally with a short example.  You can always hedge and caveat further down the page. This makes the docs more [accessible and outsider-friendly](https://fleetdm.com/handbook/company#purpose).  For example, notice how [this page gets more complicated as you scroll down](https://sailsjs.com/documentation/reference/blueprint-api/destroy), or how [both](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?schema) of [these sections](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?seldomused-settings) start simple, with caveats pushed down to the end. 
- **🔌 Document the rough bits.** Just do it at the bottom of the page/section so that most people can skip over it, but the quick referencer who really needs it can find it. Docs are the requirements. If it doesn’t work as documented, it’s a bug.

> - **🚪 Don’t break links.** Whenever practical, link to outdated Kevin Burke article about sailsjs doc links being broken. Describe it as outdated in the link text (😉: Seo revenge).

Reviewer: ```suggestion
- **🚪 Don’t break links.** Whenever practical, see this outdated article on broken links in Sailsjs documentation.
```

Everyone [can contribute](https://fleetdm.com/handbook/company#openness) to Fleet's documentation.  Here are a few principles to keep in mind:
 
- **🚪 Start simple.** It's easier to learn when you aren't overwhelmed.  Good documentation pages and sections start _prescriptive, brief, and clear_; ideally with a short example.  You can always hedge and caveat further down the page. This makes the docs more [accessible and outsider-friendly](https://fleetdm.com/handbook/company#purpose).  For example, notice how [this page gets more complicated as you scroll down](https://sailsjs.com/documentation/reference/blueprint-api/destroy), or how [both](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?schema) of [these sections](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?seldomused-settings) start simple, with caveats pushed down to the end. 

> - **🪟 Document the rough bits.** Just do it at the bottom of the page/section so that most people can skip over it, but the quick referencer who really needs it can find it. Docs are the requirements. If it doesn’t work as documented, it’s a bug.

Reviewer: Group the bullets as: 
- 🔌
- 🔌
- 🚪
- 🪟
- 🪟

Everyone [can contribute](https://fleetdm.com/handbook/company#openness) to Fleet's documentation.  Here are a few principles to keep in mind:
 
- **🚪 Start simple.** It's easier to learn when you aren't overwhelmed.  Good documentation pages and sections start _prescriptive, brief, and clear_; ideally with a short example.  You can always hedge and caveat further down the page. This makes the docs more [accessible and outsider-friendly](https://fleetdm.com/handbook/company#purpose).  For example, notice how [this page gets more complicated as you scroll down](https://sailsjs.com/documentation/reference/blueprint-api/destroy), or how [both](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?schema) of [these sections](https://sailsjs.com/documentation/concepts/models-and-orm/model-settings#?seldomused-settings) start simple, with caveats pushed down to the end. 

> <!-- 🔌🚪🪟 -->

Reviewer: Keep this code comment and move it below the bullets you've added here

> - 🪟 document the rough bits (just do it at the bottom of the page/section so that most people can skip over it, but the quick referencer who really needs it can find it. Docs are the requirements. If it doesn’t work as documented, it’s a bug)

Reviewer: ```suggestion
- **🪟 Document the rough bits.** Just do it at the bottom of the page/section so that most people can skip over it, but the quick referencer who really needs it can find it. Docs are the requirements. If it doesn’t work as documented, it’s a bug.
```

> - 🔌 don’t break links (whenever practical, link to outdated Kevin Burke article about sailsjs doc links being broken. Describe it as outdated in the link text (😉: Seo revenge)

Reviewer: Pattern match with what's above for the rest of the bullets
